src/DynamoCore/Core/CustomNodeManager.cs

```diff
@@ -1064,5 +1064,26 @@ public bool TryGetNodeInfo(string name, out CustomNodeInfo info)
             }
             return newWorkspace;
         }
+
+        public IEnumerable<Guid> GetAllDependenciesGuids(CustomNodeDefinition def)
```

Comment:
This can be marked `internal` instead. With all the issues in custom nodes, would you think we should make a test case for this? I'm guessing we can have `HomeWorkspaceModel` and then a bunch of other `CustomNodeDefinition` objects, some related and others not. Then write a test case to verify only relevant nodes are returned.